doc/sphinx/proofs/automatic-tactics/auto.rst

```diff
@@ -506,9 +506,9 @@ Creating Hints
          :undocumented:
 
    .. cmd:: Hint {| Constants | Variables } {| Transparent | Opaque } {? : {+ @ident } }
```

Comment:
You need to update the syntax here because `doc_gram_rsts` isn't able to match the old and new commands across the `{|`.  Run the command, then search `prodnCommands` to get the correct string to use here.

But first, update the edit in `common.edit_mlg` to get the commands in a single line:

```
hint: [
  :
| REPLACE "Variables" "Transparent"
| WITH [ "Constants" | "Variables" ] [ "Transparent" | "Opaque" ]
| DELETE "Variables" "Opaque"
| DELETE "Constants" "Transparent"
| DELETE "Constants" "Opaque"
```

Reply:
Thanks, I did not remember how to do this. This should all be fixed now.